tests/find_package/CMakeLists.txt

```diff
@@ -18,4 +18,5 @@ add_executable(sleigh_test
 target_link_libraries(sleigh_test PRIVATE
   sleigh::sla
   sleigh::decomp
+  sleigh::support
```

Comment:
The consequence of this change is that if you link without `support`, you get include errors if you try to include `libsleigh.hh` because the support headers include references to `std::filesystem` and other stuff.

Previously it worked fine and you'd just get a linker error if you tried to use any of the support stuff.